src/js/server/routes/apply.js

```diff
@@ -70,16 +70,67 @@ applyRouter.post('/form', auth.authenticate, applyFormUpload.single('cv'), (req,
   });
 });
 
-// The main apply page (has the login button)
+applyRouter.get('/dashboard', auth.authenticate, function(req, res) {
+  renderDashboard(req, res);
+})
+
+// The login page (has the login button)
 applyRouter.get('/', function (req, res) {
   res.render('apply/index.html');
 });
 
 // Render the form for additional applicant details
 applyRouter.get('/form', auth.authenticate, function(req, res) {
+  // TODO: If the user has already completed the form, redirect to dashboard
   renderApplyPageWithForm(res, createApplicationForm());
 });
 
+function renderDashboard(req, res) {
+  // TODO: Get these dynamically from db
+  const userHasApplied = true;
+  const userAppliedWithTeam = false;
+  const teamApplicationComplete = true;
+
+  const reviewStatus = 'rejected'; // 'rejected' or 'accepted'
+  const furtherDetailsComplete = false;
+
+  // Derive the personal application status
+  const yourApplicationStatus = userHasApplied ? 'complete' : 'incomplete';
+
+  // Derive the team application status
+  let teamApplicationStatus;
+  if (!userAppliedWithTeam)
+    teamApplicationStatus = 'na';
+  else if (!teamApplicationComplete)
+    teamApplicationStatus = 'incomplete';
+  else
+    teamApplicationStatus = 'complete';
+
+  // Derive the further application status
+  const furtherApplicationStatus = furtherDetailsComplete ? 'complete' : 'incomplete';
+
+  // Derive the overall application status
```

Comment:
Not a blocker, but something to consider when wiring up to the DB:

Do you think it would be better if this was a function in the hacker model? I see a number of benefits

- Less business logic in our route (application status becomes something we can get anywhere)
- No need for comments as the function explains what it does
- No more mutable `let`